testing/environments/snapshot.yml

```diff
@@ -5,15 +5,16 @@ services:
   elasticsearch:
     image: docker.elastic.co/elasticsearch/elasticsearch:8.0.0-7e122dd9-SNAPSHOT
     healthcheck:
-      test: ["CMD-SHELL", "curl -s http://localhost:9200/_cat/health?h=status | grep -q green"]
+      test: ["CMD-SHELL", "curl -u elastic:changeme -s http://localhost:9200/_cat/health?h=status | grep -q green"]
       retries: 300
       interval: 1s
     environment:
     - "ES_JAVA_OPTS=-Xms1g -Xmx1g"
+    - "ELASTIC_PASSWORD=changeme"
```

Comment:
Nice one, that gets us around the problem of the random password.

Reply:
Cool. 👍 This is how we do it in the apm-integration test suite and it's been fine.